packages/js/settings-editor/src/legacy/content.tsx

```diff
@@ -10,10 +17,30 @@ export const LegacyContent = ( {
 	settingsPage: SettingsPage;
 	activeSection: string;
 } ) => {
+	const section = settingsPage.sections[ activeSection ];
+
+	if ( ! section ) {
+		return null;
+	}
+
 	return (
-		<div style={ { padding: '24px' } }>
-			<p>Legacy Content: { settingsPage.label }</p>
-			<p>Active Section: { activeSection }</p>
-		</div>
+		<form>
```

Comment:
> My interpretation of that seems to indicate that there should be a <form /> around every group rather than one form around the whole group? 😅

Indeed this was the original design, but [this conversation](https://github.com/WordPress/gutenberg/issues/63624#issuecomment-2516102010) highlights that this design decision is still under discussion. I think the best way forward here will be to have a single form and handle the outcome of that question in a follow up.

><If it's meant to be a single form element around the whole set of groups then perhaps we should include a <fieldset /> [element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) around each group and <legend />

Yes thank you, great call out here. As mentioned earlier I will create a `SettingsItem` component and will make sure the output is accessibly compliant.